src/parsers/avParser/index.ts

```diff
@@ -0,0 +1,321 @@
+import * as bluebird from 'bluebird';
+import cheerio = require('cheerio');
+import * as request from 'request-promise';
+import { AV_URL, codeErrors, proxy } from '../../config/config';
+import { IAvMark } from '../../interfaces/parserInterface';
+import { ParserError } from '../../utils/errors/';
+import { transformBmvAvModel, transformMercedesAvModel } from '../../utils/parserUtils';
+import { torTriggerer } from '../../utils/torTriggerer';
+
+// tslint:disable-next-line:no-var-requires
+const Agent = require('socks5-https-client/lib/Agent');
+// tslint:disable-next-line:no-var-requires
+const _ = require('lodash');
```

Comment:
why are we doing require?

Reply:
Types for lodash don't work correctly, "Agent" types doesn't exists. And a linter swears